daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -184,6 +185,38 @@ func (l *logStream) Close() error {
 
 // create creates a log stream for the instance of the awslogs logging driver
 func (l *logStream) create() error {
+
+	// Create log group first
+	{
```

Comment:
can you extract this block into a function? I agree that having an option would be convenient. That way we don't change the default behavior.